script.js

```diff
@@ -313,12 +313,12 @@ function generateCalendar(days, elementId, startDay, disableUntil = 0) {
   for (let i = 1; i <= days; i++) {
     const dayElement = document.createElement('div');
     dayElement.textContent = i;
-    dayElement.classList.add('text-center', 'p-2', 'rounded');
+    dayElement.classList.add('text-center','text-sm', 'p-2', 'w-10', 'h-10', 'rounded', 'border-2', 'border-transparent');
```

Comment:
use an array instead

Reply:
array for the classes?
or for the "for loop"?

Reply:
done